lib/src/internal/control_client.dart

```diff
@@ -37,10 +37,10 @@ class ControlClient implements AbstractControlClient {
           options: CallOptions(metadata: {
             'cache': cacheName,
           }));
-      return CreateCacheSuccess();
+      return CreateCacheSuccess(message: "Created cache '$cacheName'");
```

Comment:
While I love the clarity and detail in these messages, I'm not sure they buy us enough to justify the added complexity in the response classes. These are really quite a lot more detailed than any of the other SDK `toString` messages, and I'm a little worried about that from a maintainability perspective. The other SDKs generally just use the response's class name for most Success and Miss cases and a combination of the response class name and a truncated display of the relevant value for Hit and some Success classes that expose values (e.g., ListLengthSuccess prints the length of the list). This allows us to implement a basic `toString` in a base `Response` class (which it looks like we'd need to add) and override `toString` in classes that need to print values, calling the parent `toString` for the class name part and just printing the values it's responsible for.

We really don't need to include any information about caches, keys, collection names, etc. and we don't output any of that info in other SDKs, so I'd much rather provide a simple default `toString` shared among all response classes and remove the requirement of constructing every response with a message. All of the information beyond the class name that needs to be conveyed by overridden `toString` methods should already be held in the subclasses that need to represent values (e.g., Hit responses).

For reference, the javascript sdk response classes demonstrate the pattern pretty clearly.